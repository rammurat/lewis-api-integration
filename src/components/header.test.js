// @ts-nocheck
import React from 'react'
import renderer from 'react-test-renderer';
import Header from './header'

import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

it('renders correctly', () => {
    const tree = renderer
        .create(<Header />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});

test('check for header class', () => {
    const wrapper = shallow(
        <Header />
    )
    expect(wrapper.hasClass('app-header')).toBe(true)
})
