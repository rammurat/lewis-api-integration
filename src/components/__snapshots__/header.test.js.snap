// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="app-header"
>
  <h5
    className="heading"
  >
    Dishwasher
  </h5>
</header>
`;
